src/plugins/autonomy/ArduPilot/ArduPilot.cpp

```diff
@@ -110,9 +110,12 @@ void ArduPilot::init(std::map<std::string, std::string> &params) {
         }
     }
 
+    mavproxy_mode_ = sc::get<bool>("mavproxy_mode", params, false);
+
     // Get parameters for transmit socket (to ardupilot)
     to_ardupilot_ip_ = sc::get<std::string>("to_ardupilot_ip", params, "127.0.0.1");
     to_ardupilot_port_ = sc::get<std::string>("to_ardupilot_port", params, "5003");
+    printf("ArduPilot: sending to udp:%s:%s\n", to_ardupilot_ip_.c_str(), to_ardupilot_port_.c_str());
```

Comment:
Can you use std::cout instead of printf? We try to limit the amount of text printed to the screen unless it is a warning or error. Can you wrap your couts in if-statements (enable_debug), where enable_debug is a plugin parameter?